javascript/javascript3/week3/index.js

```diff
@@ -0,0 +1,47 @@
+const canvas = document.getElementById('circle');
+      const window_width = window.innerWidth;
+      const window_height = window.innerHeight;
+      canvas.width = window_width;
+      canvas.height = window_height;
+      const ctx = canvas.getContext('2d');
+      function getRandomColor() {
```

Comment:
it is a bit complicated to create a random color.
check this out: 
https://stackoverflow.com/questions/1152024/best-way-to-generate-a-random-color-in-javascript/1152508